### PR TITLE
Add documentation for configuration management

### PR DESCRIPTION
**Pull request by doanac**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 0c216ab8398e85a754afca0a46ee37c7014c01d0 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/421/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 3d8eb533bbe4e22d214055492a9f5f8de9f4049b are browesable at: https://ci.foundries.io/projects/fio-docs/builds/422/docs/artifacts/html/index.html

File: source/customer-factory/configuring.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      {
+        "name": "npmtok",
+        "value": "root (this will get encrypted by tooling)",
+        "on-changed": ["/usr/bin/touch", "/tmp/npmtok-changed"]
+      },
+      {
+        "name": "A-Readable-Value",
+        "value": "This won't be encrypted and will be visible from the API",
+        "unencrypted": true
+      }
```
Comment (ricardosalveti) [inline source/customer-factory/configuring.rst:56]: This json is invalid, missing `]` here, and also missing the last EOF line so users can just do a simple copy&paste from it.

**Comment by doanac** [comment]
Docs for 09f138439646afcd9eb1427fc8429cf24fb9c11b are browesable at: https://ci.foundries.io/projects/fio-docs/builds/423/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 06b7c309274177266db96de0ace5b53ab2ace698 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/424/docs/artifacts/html/index.html

**Pull request merged**